### PR TITLE
update readme to point to fontawesome docs

### PR DESCRIPTION
**Pull request by jasonlundien**
This changes our README.md doc to point to www.fontawesome.com for setup, adding icons, etc.
I also added our code of conduct doc to the repo as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,70 +2,31 @@
...
 Refactored to support more Ruby environments with code and documentation humbly used from the excellent [bootstrap-sass](https://github.com/twbs/bootstrap-sass) project by the Bootstrap team
 
-You can find more detailed information on these changes on our [Getting Started](https://fontawesome.com/docs/web/setup/get-started) guide, our [How to Use](https://fontawesome.com/docs/web/) guide, and our [Upgrade](https://fontawesome.com/docs/web/setup/upgrade/) guide.
+---
 
-## Installation
+<!-- toc -->
 
-### Ruby on Rails
+- [Documentation](#documentation)
```
Comment (ej2) [inline README.md:13]: I don't think we need a TOC since the readme is so short.

**Review by ej2 — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
```
Comment (mateojay) [inline CODE_OF_CONDUCT.md:6]: "pledge to make" instead of "pledge to making"

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
```
Comment (mateojay) [inline CODE_OF_CONDUCT.md:42]: replace "that are not aligned" with "not aligned" so it reads "other contributions not aligned with this ...."

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
```
Comment (mateojay) [inline CODE_OF_CONDUCT.md:50]: replace "include" with "includes"

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+address, posting via an official social media account, or acting as an appointed
```
Comment (mateojay) [inline CODE_OF_CONDUCT.md:51]: Drop the comma after "account"

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at <hello@fontawesome.com>. All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
```
Comment (mateojay) [inline CODE_OF_CONDUCT.md:61]: Replace "with regard" with "about"

**Pull request merged**